src/helpers.js

```diff
@@ -43,12 +43,17 @@ export async function sendJob(worker, config) {
   config.emitKey = cryptoRandomString(10)
 
   return new Promise((resolve, reject) => {
```

Comment:
Why do we have a promise that can never resolve?

Reply:
Nevermind, resolve was hidden in folded code. 🙈

Reply:
That's what the `reject` on L57, and the `resolve` on L62 are for?